fix: clarify OSS to OSS, Cloud, Enterprise (#4548)
influxdata/docs-v2: content/influxdb/v2.4/write-data/replication/replicate-data.md

---
title: Replicate data from InfluxDB OSS
weight: 106
description: >

>   Use InfluxDB Edge Data Replication to replicate the incoming data of select

Reviewer: ```suggestion
Replicate data from select InfluxDB OSS buckets to remote buckets in InfluxDB Cloud, InfluxDB OSS, or InfluxDB Enterprise instances.
```

>   buckets from InfluxDB OSS to buckets in remote InfluxDB Cloud, InfluxDB OSS, or 
>   InfluxDB Enterprise instances.

Reviewer: ```suggestion
```

>   buckets from InfluxDB OSS to one or more buckets in either InfluxDB Cloud, 
>   InfluxDB OSS, or InfluxDB Enterprise instances.

Reviewer 2: ```suggestion
  buckets from InfluxDB OSS to buckets in remote InfluxDB Cloud,
  InfluxDB OSS, or InfluxDB Enterprise instances.
```

---

Use InfluxDB replication streams (InfluxDB Edge Data Replication) to replicate
the incoming data of select buckets to one or more buckets on a remote 
InfluxDB OSS, InfluxDB Cloud, or InfluxDB Enterprise instance.

> The following guide demonstrates how to replicate data from InfluxDB OSS to 
> InfluxDB Cloud and can also be used to replicate data to the following instances:
> - InfluxDB OSS to InfluxDB OSS
> - InfluxDB OSS to InfluxDB Enterprise

Reviewer: ```suggestion
Replicate data from InfluxDB OSS to InfluxDB Cloud, InfluxDB OSS, or InfluxDB Enterprise.
```

## Configure a replication stream

Use the [`influx` CLI](/influxdb/v2.4/tools/influx-cli/) or the
[InfluxDB {{< current-version >}} API](/influxdb/v2.4/reference/api/) to configure
a replication stream.

{{% note %}}

> To replicate data to InfluxDB OSS or InfluxDB Enterprise, adjust the desired

Reviewer: ```suggestion
To replicate data to InfluxDB OSS or InfluxDB Enterprise, adjust the 
```

---

Use InfluxDB replication streams (InfluxDB Edge Data Replication) to replicate
the incoming data of select buckets to one or more buckets on a remote 
InfluxDB OSS, InfluxDB Cloud, or InfluxDB Enterprise instance.

The following guide demonstrates how to replicate data from InfluxDB OSS to 
InfluxDB Cloud and can also be used to replicate data to the following instances:

> - InfluxDB OSS to InfluxDB OSS
> - InfluxDB OSS to InfluxDB Cloud
> - InfluxDB OSS to InfluxDB Enterprise

Reviewer 2: ```suggestion
- InfluxDB OSS to InfluxDB OSS
- InfluxDB OSS to InfluxDB Enterprise
```

or something like this, b/c you already have the the note for OSS and Enterprise:

The following guide shows how to replicate data for the following use cases:
- InfluxDB OSS to InfluxDB OSS
- InfluxDB OSS to InfluxDB Cloud
- InfluxDB OSS to InfluxDB Enterprise
...
Configure remote connection for the InfluxDB instance that you want to replicate data to.
